### PR TITLE
PPCDP-842 Initial documentation of CDP on developers.piwik.pro

### PR DESCRIPTION
**Pull request by adaqus**
Changes summary:

Added CDP to main menu:
Zrzut ekranu z 2023-05-12 12-53-14

Deprecation note in AM:
Zrzut ekranu z 2023-05-12 12-53-26

CDP docs main page:
Zrzut ekranu z 2023-05-12 12-53-51

Warning on AM deprecation in each SDK:
Zrzut ekranu z 2023-05-12 12-54-17

### CODE REVIEW

**Comment by kororokke** [comment]
I'll handle this one

**Review by kororokke — APPROVED** [review]
LGTM

File: customer_data_platform/public_api/customer_data_platform_profiles_api.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+            type: string
+            example: "3a0133748b3e524c"
+          required: true
+      responses:
+        200:
+          content:
+            application/vnd.api+json:
+              schema:
+                $ref: "#/components/schemas/ProfileAudienceIdList"
+          description: An array of last activation log entries
```
Comment (azarecki) [inline customer_data_platform/public_api/customer_data_platform_profiles_api.yaml:32]: This seems wrong

File: customer_data_platform/public_api/customer_data_platform_profiles_api.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+        400:
+          $ref: "components.yaml#/components/responses/BadRequestError"
+        402:
+          $ref: "components.yaml#/components/responses/ModuleDisabledError"
+        404:
```
Comment (azarecki) [inline customer_data_platform/public_api/customer_data_platform_profiles_api.yaml:37]: The description states `404 Resource not found (bad ID or resource deleted)` - I think we should change the description. 404 is returned when profile does not exist

File: customer_data_platform/public_api/customer_data_platform_profiles_api.yaml (hunk 1 above)
Comment (adaqus) [reply customer_data_platform/public_api/customer_data_platform_profiles_api.yaml]: Changed. Also changed in original docs: https://github.com/PiwikPRO/PPMS-CDP/pull/341

File: customer_data_platform/public_api/customer_data_platform_profiles_api.yaml (hunk 2 above)
Comment (adaqus) [reply customer_data_platform/public_api/customer_data_platform_profiles_api.yaml:37]: Also changed in original docs: https://github.com/PiwikPRO/PPMS-CDP/pull/341

**Pull request merged**